src/index.css

```
@@ -45,18 +45,33 @@
 }
 
 body {
-  @apply antialiased bg-[#1b2320] text-emerald-50 font-sans;
+  @apply antialiased bg-forest-bg text-forest-text font-sans;
 }
 
 @theme inline {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)